sdk/python/generative-ai/rag/notebooks/azure_cognitive_search/acs_mlindex_with_langchain.ipynb

```diff
@@ -437,12 +437,7 @@
     "    )\n",
     "    use_automatic_compute(crack_and_chunk)\n",
     "\n",
-    "    # `generate_embeddings_parallel` uses AzureML an parallel job to process batches of chunks in parallel.\n",
-    "    # By default 3 files are processed in a single batch, all chunks from each file are included.\n",
-    "    # For larger numbers of files more instances can be added using the `instance_count` parameter below so batches can be processed in parallel.\n",
-    "    # When using AOAI, the maximum number requests per-minute is usually the limiting factor.\n",
-    "    # To read more about parallel jobs see: https://learn.microsoft.com/en-us/azure/machine-learning/how-to-use-parallel-job-in-pipeline?view=azureml-api-2&tabs=cliv2\n",
-    "    generate_embeddings = generate_embeddings_parallel_component(\n",
+    "    generate_embeddings = generate_embeddings_component(\n",
```

Comment:
also need to update this file - https://github.com/Azure/azureml-examples/blob/d18f338467d3300a3bb143edc1fa6ec8e6b14d23/sdk/python/generative-ai/rag/notebooks/mlindex_with_testgen_autoprompt.ipynb